pkgs/applications/networking/mullvad/mullvad.nix

```diff
@@ -13,29 +13,18 @@
 , openvpn-mullvad
 , shadowsocks-rust
 }:
-let
```

Comment:
This is quite ideal, very glad they made this change upstream.

I'll be able to test this some time tomorrow.